tests/e2e/app-elements/item-details.component.ts

```diff
@@ -119,6 +119,17 @@ export class ItemDetails {
         expect(customFieldContent.trim()).toBe(fieldValue);
     }
 
+    static async VerifyCustomTreeNodeComponent(){
+        await BrowserWaitForElement(ItemDetailsMap.AddRelatedDataNewsItemsButton);
+        await ItemDetailsMap.AddRelatedDataNewsItemsButton.click();
+        await BrowserWaitForElement(ItemDetailsMap.FirstRelatedDataItem);
+        const createdOn = await ItemDetailsMap.FirstRelatedDataItem.findElement(by.css("span[data-sftest=\"custom-created-by\"]"));
```

Comment:
This is failing on the CI with findElement is not a function. Use element instead

Reply:
thank you, i changed it